src/amber/cli/commands/exec.cr

```diff
@@ -6,8 +6,14 @@ module Amber::CLI
 
     class Exec < Command
       command_name "exec"
-      @filename : String = "./tmp/#{Time.now.epoch_ms}_console.cr"
-      @filelogs : String = @filename.sub("console.cr", "console_result.log")
+      @filename : String
+      @filelogs : String
+
+      def initialize(__previous, __argv)
```

Comment:
Why not use `*args` and `super`?

Reply:
It was the minimum change. But it should be equivalent.